testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/page/SigningInPage.java

```diff
@@ -192,6 +192,15 @@ public String getUserLabel() {
             return getTextFromItem(LABEL);
         }
 
+        public boolean hasCreatedAt() {
+            boolean result = false;
+            try {
+                result = getItemElement(CREATED_AT) != null;
+            } catch (NoSuchElementException e) {}
+
+            return result;
```

Comment:
This is just a code style / optimization suggestion. I'll leave it up to you if you change it.

No need to create another variable. You can use something like:
```suggestion
            try {
                return getItemElement(CREATED_AT).isDisplayed();
            } catch (NoSuchElementException e) {
                return false;
            }
```